server/src/index.js

```diff
@@ -1,12 +1,15 @@
 import '@babel/polyfill';
 import express from 'express';
+import bodyParser from 'body-parser';
 import db from './models/index';
 import routes from './routes/index';
 
 
 const app = express();
 
 app.use(express.json());
+// support parsing of application/x-www-form-urlencoded post data
+app.use(bodyParser.urlencoded({ extended: true }));
```

Comment:
For the body-Parser, I thought we settled for using the express.json instead. They do the same thing so we excluded importing body-parser.